ubr/main.py

```diff
@@ -1,61 +1,60 @@
 import argparse
 import os, sys
 from os.path import join
-from conf import logging
+import logging
 from ubr import conf, utils, s3, mysql_target, file_target, tgz_target, psql_target
 from ubr.descriptions import load_descriptor, find_descriptors, pname
 
 LOG = logging.getLogger(__name__)
-_handler = logging.FileHandler('ubr.log')
-_handler.setFormatter(conf._formatter)
-LOG.addHandler(_handler)
-LOG.setLevel(logging.INFO)
-
-TARGETS = {
-    'backup': {
-        'files': file_target.backup,
-        'tar-gzipped': tgz_target.backup,
-        'mysql-database': mysql_target.backup,
-        'postgresql-database': psql_target.backup,
-    },
-
-    'restore': {
-        'files': file_target.restore,
-        'tar-gzipped': tgz_target.restore,
-        'mysql-database': mysql_target.restore,
-        'postgresql-database': psql_target.restore,
-    }
-}
-
-def do(action, target, args, destination):
-    # print 'doing',action,'on',target,'with',args,'at',destination
-    if action not in TARGETS.keys():
-        LOG.warn("unknown action %r - I only know how to do %s", action, ", ".join(TARGETS.keys()))
-        return None
-    if target not in TARGETS[action].keys():
-        LOG.warn("unknown target %r - I only know about %s", target, ", ".join(TARGETS[action].keys()))
-        return None
-    return TARGETS[action][target](args, destination)
+
+#
+# utils
+#
+
+def getmod(target):
```

Comment:
I think I was being too clever with the environmenterror, switched it to a plain `assert`